src/components/LetterForm.tsx

```diff
@@ -35,7 +35,7 @@ export function LetterFormDialogContent() {
         content,
         type,
       },
-      interaction_data: { numDrags: 0, numOpens: 0 } as LetterInteractionData,
+      interaction_data: { numDrags: 0, numOpens: 0 } satisfies LetterInteractionData,
```

Comment:
wait this has a type error i think you can only use this when defining types

Reply:
incredibly sad this only works at the definition level and not nested in objects